validBST.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var isValidBST = function(root, lowerBound = -Infinity, higherBound = Infinity) {
+
+  if (!root) return true;
+
+    if (root.val >= higherBound || root.val <= lowerBound) return false;
+
+   return (!isValidBST(root.left, lowerBound, root.val) || !isValidBST(root.right, root.val, higherBound))
```

Comment:
To simplify try to use De Morgan law:
![image](https://user-images.githubusercontent.com/10159037/69519516-ed42a180-0f62-11ea-9fd2-cf40999cd8ad.png)

Reply:
Now, it works slower than in the previous one.

Reply:
How so?